### DISCUSSION TITLE
分享一份贵州省贵阳市乌当区中天吾乡烂尾楼业主判决书部分内容

### DISCUSSION THREAD

**Discussion by MasterZhang007** [WeNeedHome/SummaryOfLoanSuspension#1204] [Show and tell]
![image](https://user-images.githubusercontent.com/21271458/229723034-d47488b5-56a7-46bd-a6fe-d15c5a23f781.png)

这是一位中天吾乡烂尾楼业主的判决书部分截图，项目已停工，荒山晾在那里，什么都没有动，但是得不到fa yuan的支持。

本人于2020年5月购买了贵州省贵阳市乌当区中天吾乡B2组团的一套房子，合同约定2022年9月1日交房。如今交房日期已逾半年，B2组团仍是一片荒山。

我也想起诉开发商解除合同，然后与银行协商停贷，但是看了其他业主的判决结果，感觉此事毫无胜算。

**Comment by Sunny79928** [comment]
走诉讼如果能解决的话 ，大家都会去诉讼。问题是司法能凌驾于00之上么?

**Comment by Cegazhos** [comment]
有成功的案例，也有失败的案例。
中国是公检法，不是法检公。具体是否胜诉得先看公的情况。公没钱，或者有利益关系，那很难。
听说过贵州有很多债务，应该是很难在贵州这个地方胜诉了。

**Comment by dinglino1** [comment]
烂尾楼能不能理解为就是堂而皇之的转嫁当地债务，让老百姓分摊。
